js/options.js

```diff
@@ -31,7 +32,7 @@
       selector: '#font-size',
       value: 'value',
       decode: id,
-      render: set('#code', 'style.fontSize', id)
+      render: set('#code', 'style.fontSize', px)
```

Comment:
As mentioned before, I'd rather have the unit come with the value, so we could use id here, no?